src/js/components/List/index.d.ts

```diff
@@ -61,6 +61,7 @@ export interface ListProps<ListItemType> {
   onOrder?: (orderedData: ListItemType[]) => void;
   pad?: PadType;
   paginate?: boolean | PaginationType;
+  pinned?: string[];
```

Comment:
I think this would need to be `pinned?: (string | number)[];` to align with the return value of `itemKey()`.

Reply:
Good point, I'll make that update. I originally copied the disabled type definition.